ui.content/src/main/content/jcr_root/content/aemtraining/us/en/.content.xml

```diff
@@ -5,79 +5,102 @@
         cq:contextHubPath="/etc/cloudsettings/default/contexthub"
         cq:contextHubSegmentsPath="/etc/segmentation/contexthub"
         cq:template="/conf/aemtraining/settings/wcm/templates/content-page"
+        jcr:mixinTypes="[cq:LiveRelationship,cq:LiveSync]"
```

Comment:
How is this file different from [ui.content/src/main/content/jcr_root/content/aemtraining/language-masters/en/.content.xml](https://github.com/Zubayr99/AEMDemo/pull/8/files#diff-d05b1b856cebd0026b472067794c9936a55504e831cae88c3d5b2683dc59c6bb) ? Why is it here?